### ISSUE TITLE
no_proxy

### ISSUE THREAD

**Issue by joaquin386** [np-maintain/global-tunnel#61]
Hi,

Which is the no_proxy env variable you read ...... `no_proxy` cause you have `https_proxy` and `http_proxy` but you never mention `no_proxy`.

BR

Jo

**Comment by rvalle** [comment]
yes, any node service will have both lan connection and wan connections no_proxy is a must.
